### DISCUSSION TITLE
Nuxt internals: Where/when are nuxt plugins loaded?

### DISCUSSION THREAD

**Discussion by aptiko** [nuxt/nuxt#9807] [Questions (Nuxt 2)]
Is there any text that describes the nuxt starting process? Here's a variety of questions I'm struggling with: What does `loadNuxt()` do? What does `build()` do? When are modules imported? When are they run? When are nuxt plugins loaded? Which place in the Nuxt code imports/executes the nuxt plugins? What does `@nuxt/vue-app` do?

**Comment by atinux** [comment]
This is definitely something we shall work on to explain in our documentation. Cc @danielroe
